### ISSUE TITLE
Wrong window placement in Quake mode with two monitors

### ISSUE THREAD

**Issue by edubxb** [gnunn1/terminix#603]
When Quake mode is not set to use the whole screen width, with two monitors, the terminix window placement is wrong.

My monitors have different resolutions:

1920x1080 left, and 2560x1440 right

![screenshot from 2016-12-12 12-16-33](https://cloud.githubusercontent.com/assets/1192339/21097522/262561ea-c065-11e6-9064-a761454d3a0f.png)

My current config:
![screenshot from 2016-12-12 11-28-38](https://cloud.githubusercontent.com/assets/1192339/21097505/146504e2-c065-11e6-9828-add7451bc938.png)

**Comment by gnunn1** [comment]
What version of terminix are you using?

**Comment by edubxb** [comment]
```terminix --version
Versions
	Terminix version: 1.3.5
	VTE version: 0.46
	GTK Version: 3.22.4

Terminix Special Features
	Notifications enabled=0
	Triggers enabled=0
	Badges enabled=0```

**Comment by gnunn1** [comment]
I believe this has been fixed in the latest version as I cannot reproduce it on an identical setup.

**Comment by edubxb** [comment]
I will try with a newer version.

**Comment by edubxb** [comment]
Fixed in 1.4.0-rc1.
